lib/addons/amp/index.ts

```diff
@@ -60,7 +48,7 @@ export const enum DeploymentMode {
  * Defaults options for the add-on
  */
 const defaultProps = {
-    workspaceName: 'blueprints-amp-workspace',
+    ampPrometheusEndpoint: "ws-sample-workspace",
```

Comment:
Why do we need this attribute here? In the past it was needed to create the workspace, but that approach change.  Is it one of those cases, when customers should really pass the endpoint to the addon to make it work?

Reply:
Fixed.